bot.py

```diff
@@ -13,7 +13,7 @@
 from discord.ext import commands
 from typing_extensions import Self
 
-from .context import AloneContext
+from utils.context import AloneContext
```

Comment:
Couldn't you set it just as from utils import AloneContext here? for ex L..13 ext/error.py